website/www/site/content/en/contribute/_index.md

```diff
@@ -21,57 +22,65 @@ See the License for the specific language governing permissions and
- - write new examples
- - improve your favorite language SDK (Java, Python, Go, etc)
- - improve specific runners (Apache Flink, Apache Spark, Google
-   Cloud Dataflow, etc)
- - improve or add IO connectors
- - add new transform libraries (statistics, ML, image processing, etc)
- - work on the core programming model (what is a Beam pipeline and how does it
-   run?)
- - improve the developer experience (for example, Windows guides)
- - add answers to the [contribution FAQ](
- https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
- - organize local meetups of users or contributors to Apache Beam
+- ask or answer questions on [user@beam.apache.org](/community/contact-us/) or
+  [stackoverflow](https://stackoverflow.com/questions/tagged/apache-beam)
+- review proposed design ideas on [dev@beam.apache.org](/community/contact-us/)
+- file [bug reports](https://issues.apache.org/jira/projects/BEAM/issues)
+- review [changes](https://github.com/apache/beam/pulls)
+- work on the core programming model (what is a Beam pipeline and how does it
+  run?)
+- improve the developer experience (for example, Windows guides)
+- organize local meetups of users or contributors to Apache Beam
 
-Most importantly, if you have an idea of how to contribute, then do it!
+</div>
+
+...and many more. Most importantly, if you have an idea of how to contribute, then do it!
+
+</br>
 
 ## Contributing code
 
+</br>
+
 Below is a tutorial for contributing code to Beam, covering our tools and typical process in
 detail.
 
+</br>
+
 ### Prerequisites
 
-To contribute code, you need
+<div class="arrow-list">
+
+- a GitHub account
+- a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
+- [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally
+- [Go](https://golang.org) 1.12 or later installed for Go SDK development
+- Python 3.6, 3.7, and 3.8. Yes, you need all four versions installed.
+
+  pip, setuptools, virtualenv, and tox installed for Python development
+
+- for large contributions, a signed [Individual Contributor License
+  Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+  Software Foundation (ASF).
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
- - Python 3.6, 3.7, and 3.8. Yes, you need all three versions installed.
-    - pip, setuptools, virtualenv, and tox installed for Python development
- - for large contributions, a signed [Individual Contributor License
-   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
-   Software Foundation (ASF).
+</div>
```

Comment:
Any changes to the actual content of the page should be done on master, instead of on the website-revamp branch

Reply:
I completely agree with that, but other choices are: 
1. to change all lists globally to arrow ones, but it could be troublesome, because in app we have lists which we don't want to look like this.
2. set arrow lists and spacings after each paragraph with one wrapper that way we will have less ` <div>`s, but more complex classes for each html tag
3. take all lists to shortcodes and content to yamls in data folder, the markdown file would be cleaner, but maintenance of the code would be harsher, because to change one page we would need to change a couple of files

About number of `<br>`s, I can drop it down, adding some spacings to the "arrow-list" class.
If we are talking about the content of pages, some are just shorter versions, but some of them are like complete different pages like for example "Become a Beam Committer" page, it has totally different approach, it would be impossible to commit firstly only content to master branch and then add styles with merge of the `website-revamp `

Reply:
Would it be possible to just use a different layout on the "become a committer" page that renders lists with arrow-list?

Reply:
I feel very strongly that we shouldn't be making changes to the website content in the `website-revamp` branch. If we do that, we're back in the same predicament that we had with the switch to hugo where we had to pause changes to the website to avoid merge conflicts.

@bntnam you mentioned in an offline conversation that if content changes were required you would have a plan for it - do you have any suggestions here?

Reply:
> Would it be possible to just use a different layout on the "become a committer" page that renders lists with arrow-list?

Yes, I can make custom layouts for single pages, I think you mean the "Contribution guide" page right? Because there are this arrow-lists.

Reply:
Whoops yes I meant the contribution guide, sorry about that.

Reply:
I pushed version with one wrapper setting arrow lists

Reply:
I think that we could remove this div, like @TheNeuralBit requested, by styling `<li>` elements with some wrapper class in `/layouts/contribute/baseof.html`. We could replace 
```
<div class="body__contained body__section-nav">
      {{ .Content }}
</div>
```
with
```
<div class="body__contained body__section-nav arrow-list">
      {{ .Content }}
</div>
```

Reply:
I made custom layout for this page and minimalized number of code in div's

Reply:
and also deleted most of `br's `, some stayed in contribute/_index.md, but this page is pretty uniq